include/SFML/Window/Mouse.hpp

```diff
@@ -58,6 +59,8 @@ class SFML_WINDOW_API Mouse
 
         Count       ///< Keep last -- the total number of mouse buttons
     };
+    
+    MML_COMPAT_ALIAS(ButtonCount, Button::Count)
```

Comment:
Let's bring back ButtonCount into Button enum, maybe? That way, we can remove MML_COMPAT_ALIAS for now and make things simpler. We're unlikely to need another "aliases" like this.